lib/routes/try.js

```diff
@@ -2,6 +2,30 @@ var express = require('express');
 var request = require('request');
 var streamBuffers = require("stream-buffers");
 
+function getRequestedUrl( req ) {
+  var url = req.headers['x-request-url'];
+  if (!url) {
+    return resp.status(500).send('URL must be provided in x-request-url header');
+  }
+  delete req.headers['x-request-url'];
+  return url;
+}
+
+function serializeHeaders( req ) {
+  var array = [];
+  Object.keys(req.headers).forEach( function( key ) {
+    var value = req.headers[key];
+    array.push( key + ': ' + value );
+  });
+  return array.join('\n');
+}
+
+function enrichResponseWithRequestHeaders( req, resp ) {
+  var serializedHeaders = serializeHeaders( req );
```

Comment:
I think it's so the client knows exactly what headers were sent with the `request` module, rather than those the browser sends (as we don't really want the browser's content type etc making it through). 
@lfryc a comment in the code here describing what it's doing might be good as i'm not 100% sure either